.travis.yml

```diff
@@ -0,0 +1,87 @@
+# Travis configuration. Run FI hardening tests.
+
+addons:
+  apt:
+    packages:
+      - "python3-pip"
+
+language: rust
```

Comment:
rust is not required anymore, so this line can be removed.

Reply:
Will remove unnecessary stuff